src/lib/containers/ModalDownload.tsx

```diff
@@ -90,6 +90,7 @@ export default class ModalDownload extends React.Component<
       writeHeader,
       includeRowIdAndRowVersion,
       csvTableDescriptor: { separator },
+      additionalFilters: queryRequest.query.additionalFilters
```

Comment:
Typically I'd want to see a test that verifies that this parameter gets included but the existing ModalDownload tests are not in good shape, so no need right now.